tests/tests_unit/test_data_classes/test_capabilities.py

```diff
@@ -135,7 +135,7 @@ def unknown_cap_with_extra_key():
 class TestCapabilities:
```

Comment:
Let's add some test cases around this? unknown scopes/actions/acls

Reply:
Yeah, just waiting to see if you guys had a better idea for implementation 😅

Reply:
...done!